test/sdk-benchmarks/src/main/java/software/amazon/awssdk/benchmark/utils/BenchmarkConstantV2.java

```diff
@@ -0,0 +1,480 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.benchmark.utils;
+
+public class BenchmarkConstantV2 {
```

Comment:
How is this different from BenchmarkConstant? Should we add constants there or rename this class to something that doesn't imply BenchmarkConstant is deprecated in favor of this?

Reply:
`BenchmarkConstant` defines constants for `AllTypes` types, for the `successfulResponse` test. This one defines a different set of constants for the marshalling test.

Reply:
Can we rename this to JsonMarshallingConstant or something like that? My main concern is that this may imply the existing BenchmarkConstant is deprecated, which doesn't seem to be the case.